practice_06.08.2019_System.IO/Task3_FileCharacters/FileCharacters.cs

```diff
@@ -0,0 +1,65 @@
+﻿using practice_06._08._2019_System.IO.Task2_FileCreation;
+using System;
+using System.IO;
+
+//Разработайте приложение, которое позволяет посчитать сколько содержится в файле символов, строк и слов.
+//Разделителями слов считать пробелы и перенос строки.
+
+namespace practice_06._08._2019_System.IO.Task3_FileCharacters
+{
+    public class FileCharacters
+    {
+        public void CountCharacters(string filePath)
+        {
+            string tempText = "";
```

Comment:
Please use string.Empty instead of empty literals.

Reply:
fixed